src/core/algorithms/fd/sfd/cords.cpp

```diff
@@ -67,6 +67,8 @@ void Cords::RegisterOptions() {
     RegisterOption(
             Option{&max_amount_of_categories_, kMaxAmountOfCategories, kDMaxAmountOfCategories}
                     .SetValueCheck(check_positive));
+
+    RegisterOption(Option{&fixed_sample_, kFixedSample, kDFixedSample});
```

Comment:
Provide the default value to the option
```suggestion
    RegisterOption(Option{&fixed_sample_, kFixedSample, kDFixedSample, false});
```